src/Raven.Server/Documents/Handlers/DocumentReplicationHandler.cs

```diff
@@ -1,16 +1,190 @@
-﻿using System.Threading.Tasks;
+﻿using System.Net;
+using System.Threading.Tasks;
 using Raven.Server.Routing;
-using Raven.Server.Web;
+using Raven.Server.ServerWide.Context;
+using Sparrow.Json;
+using Sparrow.Json.Parsing;
+using System.Linq;
 
 namespace Raven.Server.Documents.Handlers
 {
-    public class DocumentReplicationRequestHandler : RequestHandler
+    public class DocumentReplicationtHandler : DatabaseRequestHandler
     {
         [RavenAction("/databases/*/replication/topology", "GET")]
         public Task GetReplicationTopology()
         {
             HttpContext.Response.StatusCode = 404;
             return Task.CompletedTask;
         }
+
+        [RavenAction("/databases/*/replication/active-connections", "GET")]
+        public Task GetReplicationActiveConnections()
+        {
+            HttpContext.Response.StatusCode = (int)HttpStatusCode.OK;
+            DocumentsOperationContext context;
+            using (ContextPool.AllocateOperationContext(out context))
+            using (var writer = new BlittableJsonTextWriter(context, ResponseBodyStream()))
+            {
+                var incoming = new DynamicJsonArray();
+                foreach (var item in Database.DocumentReplicationLoader.IncomingConnections)
+                {
+                    incoming.Add(new DynamicJsonValue
+                    {
+                        ["SourceDatabaseId"] = item.SourceDatabaseId,
+                        ["SourceDatabaseName"] = item.SourceDatabaseName,
+                        ["SourceMachineName"] = item.SourceMachineName,
+                        ["SourceUrl"] = item.SourceUrl
+                    });
+                }
+
+                var outgoing = new DynamicJsonArray();
+                foreach (var item in Database.DocumentReplicationLoader.OutgoingConnections)
+                {
+                    outgoing.Add(new DynamicJsonValue
+                    {
+                        ["Url"] = item.Url,
```

Comment:
When connected? What is the last sent etag?
Are we gossiping or actively sending?